Description: Remove unwanted compiler flags
 * -Werror flag is removed because of fatal warnings in Qt
 * Debian packages are compiled with second optimization level
 * Disable mandatory LTO
   - This is temporary workaround to avoid FTBFS on mips64el. For other
     architectures LTO is enabled by the d/rules file.
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: 2017-09-14

--- a/Telegram/gyp/settings_linux.gypi
+++ b/Telegram/gyp/settings_linux.gypi
@@ -24,7 +24,6 @@
         'linux_common_flags': [
           '-pipe',
           '-Wall',
-          '-Werror',
           '-W',
           '-fPIC',
           '-Wno-unused-variable',
--- a/Telegram/gyp/telegram_linux.gypi
+++ b/Telegram/gyp/telegram_linux.gypi
@@ -76,16 +76,11 @@
     'configurations': {
       'Release': {
         'cflags_c': [
-          '-Ofast',
           '-fno-strict-aliasing',
         ],
         'cflags_cc': [
-          '-Ofast',
           '-fno-strict-aliasing',
         ],
-        'ldflags': [
-          '-Ofast',
-        ],
       },
     },
     'conditions': [
@@ -95,13 +90,6 @@
         'cflags_c': [ '-g' ],
         'cflags_cc': [ '-g' ],
         'ldflags': [ '-g' ],
-        'configurations': {
-          'Release': {
-            'cflags_c': [ '-flto' ],
-            'cflags_cc': [ '-flto' ],
-            'ldflags': [ '-flto' ],
-          },
-        },
       }, {
         'ldflags': [
           '-Wl,-wrap,__divmoddi4',
